### DISCUSSION TITLE
Unable to find RTSP URL

### DISCUSSION THREAD

**Discussion by Gohab94** [tsightler/ring-mqtt#869] [Support Q&A]
![Capture](https://github.com/tsightler/ring-mqtt/assets/174978588/8ddb191e-c06a-4f40-a989-8083dcc5cf97)
I cannot find the RTSP URL when searching "source" or "RTSP" in Notepad after downloading the diagnostic info from the link shown above. Ring and Ring-MQTT are downloaded. Any help would be great.

![2](https://github.com/tsightler/ring-mqtt/assets/174978588/cde0bc3b-65fc-403b-98e8-4cff973000f0)
![3](https://github.com/tsightler/ring-mqtt/assets/174978588/5508955c-beb4-4ed4-bb9d-3a6f77bd5e7f)
Ring-MQTT is showing up under addons but not itergrations, not sure if that's the issue...?

Iv been attempting to follow these instructions; https://community.home-assistant.io/t/live-view-of-ring-doorbell-camera/428933/10

**Comment by jeffgraham-hub** [comment]
Enable the Info entity under diagnostics and look at its attributes.
![Screenshot_20240707_223319_Home Assistant](https://github.com/tsightler/ring-mqtt/assets/65316307/67bb25cc-fcb0-456b-a176-88173b59fc82)
![Screenshot_20240707_223413_Home Assistant~2](https://github.com/tsightler/ring-mqtt/assets/65316307/a9013d8f-facb-4c6e-9b49-f99f9e309c91)

**Comment by Gohab94** [comment]
I'm a complete noob here,i'm not see what you seeing... where do I go to do this?

**Comment by Gohab94** [comment]
![4](https://github.com/tsightler/ring-mqtt/assets/174978588/17abc5be-37f4-402b-b34a-9555d337c936)
Nothing else under diagnostics for me :/

**Comment by tsightler** [comment]
@Gohab94  The screenshots you have posted are for the native HA Ring integration, not for Ring-MQTT.   Ring-MQTT is an addon, not a native integration and cameras integrate via MQTT.  I do not see in your screenshots any indication that that you have configured MQTT for your Home Assistant instance so there's zero chance that it is working.

**Reply by Gohab94** [reply]
> @Gohab94 The screenshots you have posted are for the native HA Ring integration, not for Ring-MQTT. Ring-MQTT is an addon, not a native integration and cameras integrate via MQTT. I do not see in your screenshots any indication that that you have configured MQTT for your Home Assistant instance so there's zero chance that it is working.
![Capture](https://github.com/tsightler/ring-mqtt/assets/174978588/b62a7e25-3e4d-4689-bcb0-9868b0645a07)


@tsightler How would I go about configuring Ring-MQTT?

**Reply by tsightler** [reply]
It's not about configuration ring-mqtt, rather, ring-mqtt depends on you having previously configured MQTT in Home Assistant, i.e. installing Mosquitto addon, configuration Mosquitto integration, etc.  It's quite simple, pretty much just accept defaults everywhere, same for ring-mqtt, and it will work, but ring-mqtt can't work if you haven't previosuly configured MQTT itself.  It's explicitly mentioned that having a working MQTT integration is a pre-requisite for ring-mqtt (as it should probably be implied by the name as well).  The setup of MQTT/Mosquitto for Home Assistant is documented a few hundred times on the internet already, including in the official HA documentation, so I'm not going to repeat it here.
